src/services/bulkstatus.service.js

```diff
@@ -140,6 +140,26 @@ async function checkBulkStatus(req, res) {
               };
               response.entry[0].resource.parameter.push(inputResult);
             });
+            const successCountResult = {
+              name: 'outcome',
+              part: [
+                { name: 'associatedInputUrl', valueUrl: url.valueUrl },
+                {
+                  name: 'operationOutcome',
+                  resource: {
+                    resourceType: 'OperationOutcome',
+                    issue: [
+                      {
+                        severity: 'information',
+                        code: 'informational',
+                        details: { text: `Successfully processed ${ndjsonStatus.successCount} rows.` }
```

Comment:
When a file failed to load (ex. the bulk server is not running). This code is still reached and creates this entry with `"Successfully processed null rows."`. This outcome shouldn't be made if the file was unable to be read at all.